### ISSUE TITLE
Move add_interesting_values to EntitySet

### ISSUE THREAD

**Issue by gsheni** [alteryx/featuretools#1262]
- The `add_interesting_values` function goes through each of series in the DataFrame, and adds interesting values to the Variable.
- With the Woodwork integration, this function should be moved to the EntitySet.

**Comment by thehomebrewnerd** [comment]
@gsheni @rwedge Any thoughts on how we want the API to work here since there is already an `add_interesting_values` method on `EntitySet`? We could add an optional parameter to the existing method to specify which entities to add interesting values for, and if it not specified, interesting values will be added for all entities. 

Something like this:
```python
# Add interesting values for all entities - same as current behavior for EntitySet.add_interesting_values()
es.add_interesting_values()

# Add interesting values for the 'customer' entity - same result as calling Entity.add_interesting_values() on the
# customer Entity
es.add_interesting_values(entity_id='customer')
```

**Comment by gsheni** [comment]
@thehomebrewnerd My vote would be for what you suggested functionality. I'm not sure if we still want to reference parameters/docs with Entity or if those should be changed to DataTable. (In this case `entity_id` -> `datatable_id`)

**Comment by thehomebrewnerd** [comment]
I think eventually we would switch this to datatable, but we can still make this update now and use it with the existing Entity objects until we have fully replaced Entity with a DataTable. I guess it just depends on if we want to merge in these small pieces first, or merge them as part of a larger update to use Woodwork.

**Comment by thehomebrewnerd** [comment]
Closed by #1269
